KTL.js

```diff
@@ -5619,7 +5621,28 @@ function Ktl($, appInfo) {
                 }
             }
         }
-
+        function makeTableHeaderSticky(viewId, keywords, data) {
+            const kw = '_sth';
+            let numOfRecords = 10;
+            let viewHeight = 800;
+            if (keywords[kw] && keywords[kw][0]?.params && keywords[kw][0]?.params[0].length) {
```

Comment:
@cortexrd Do you still have the constraint of optional chaining? If so, '?.' will not do.

Reply:
Ok I didn't realise that shall I change it?